config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -1795,8 +1795,8 @@ spec:
       name: resizer
     - image: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.13.0
       name: externalhealthmonitorcontroller
-    - image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.9.0
+    - image: quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.10.0
       name: metadataretriever
   skips:
-    - dell-csm-operator.v1.6.0
-  version: 1.7.0
+    - dell-csm-operator.v1.8.0
+  version: 1.8.0
```

Comment:
Why did it previously have a mismatch (1.6/1.7) but it's got parity between the two in 1.8?

Reply:
fixed